pkg/storaged/storage.scss

```diff
@@ -219,6 +219,10 @@ div.progress {
     min-width: 250px;
 }
 
```

Comment:
When we don't show the headers then the first item does not have a top border. I don't really consider it a bug. How do you think we should override the default behavior here?

Reply:
All the table demos have headings; I guess non-heading isn't used as much?
https://www.patternfly.org/v4/components/table

Data list demos have lines at the top
https://www.patternfly.org/v4/components/data-list

I think it's a bug or an oversight.

We should probably fix it elsewhere though? Perhaps in PF4 overrides and file an issue upstream? WDYT?

Reply:
No idea if it's a bug. Sounds like an easy however.

Reply:
Oh, but we're not applying it to PF4 widgets here; this is all legacy code. But I think it's also a problem with PF4 too.

Reply:
We use actually directly PF4 flex.